### PR TITLE
Pin pint to 0.20

### PR DESCRIPTION
**Pull request by Illviljan**
Newest pint crashes our tests for some reason, pin it for now. 

- [ ] Closes #xxxx
- [ ] Tests added
- [ ] User visible changes (including notable bug fixes) are documented in `whats-new.rst`
- [ ] New functions/methods are listed in `api.rst`

### CODE REVIEW

**Comment by Illviljan** [comment]
Indeed is pint.

File: ci/requirements/environment-py311.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ dependencies:
   - numpy
   - packaging
   - pandas
-  - pint
+  - pint<0.20
```
Comment (keewis) [inline ci/requirements/environment-py311.yml:31]:
```suggestion
  - pint<0.21
```

File: ci/requirements/environment-windows-py311.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ dependencies:
   - numpy
   - packaging
   - pandas
-  - pint
+  - pint<0.20
```
Comment (keewis) [inline ci/requirements/environment-windows-py311.yml:28]:
```suggestion
  - pint<0.21
```

File: ci/requirements/environment-windows.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@ dependencies:
   - numpy<1.24
   - packaging
   - pandas
-  - pint
+  - pint<0.20
```
Comment (keewis) [inline ci/requirements/environment-windows.yml:28]:
```suggestion
  - pint<0.21
```

File: ci/requirements/environment.yml [hunk 4]
```diff
@@ -28,7 +28,7 @@ dependencies:
   - numpy<1.24
   - packaging
   - pandas
-  - pint
+  - pint<0.20
```
Comment (keewis) [inline ci/requirements/environment.yml:31]:
```suggestion
  - pint<0.21
```

**Review by keewis — COMMENTED** [review]
pinning sounds good to me, but the new version is `0.21`

**Pull request merged**